pkgs/development/python-modules/monkeytype/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  libcst,
+  mypy-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "monkeytype";
+  version = "23.3.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Instagram";
+    repo = "MonkeyType";
+    rev = "v${version}";
+    hash = "sha256-DQ/3go53+0PQkhZcL2dX8MI/z4Iq7kTYd5EbacMNxT4=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
```

Comment:
```suggestion
```
Added automatically by `nix-init` but not needed.
-> Same for the other two packages.